pyproject.toml

```diff
@@ -175,4 +176,8 @@ max-complexity = 30
 
 [tool.ruff.flake8-tidy-imports]
 [tool.ruff.flake8-tidy-imports.banned-api]
-"logging".msg = "The logging framework is banned. Please use demisto_sdk.commands.common.logger."
+# "logging".msg = "The logging framework is banned. Please use demisto_sdk.commands.common.logger."
```

Comment:
why logging is removed?

Reply:
it was configured but not enabled (under `select`, above), so it never really ran.

Now, when I enable the rule, it fails in 20+ places on `logging`.
Opened [CIAC-7433](https://jira-hq.paloaltonetworks.local/browse/CIAC-7433) for that.